apps/explorer/config-default.yaml

```diff
@@ -0,0 +1,317 @@
+#       - For redefining components, or for more information, follow the instructions in ./README.md
+#   ⚙️Read more here:
+#       https://github.com/gnosis/gp-v1-ui/wiki/Config
+
+# App Id
+#   Identify the app. The IDs range from 1 until 256
+#   Every transaction sent to the blockchain will include the ID and some basic analytic info
+#
+#   ⚠️IMPORTANT: The ID must be unique per App/Client of Gnosis Protocol:
+#       https://github.com/gnosis/gp-v1-ui/wiki/App-Ids-for-Forks
+#   More information about forking the web:
+#       https://github.com/gnosis/gp-v1-ui/wiki/Fork-project
+#
+#appId: 1
+
+# Name of the app
+name: Gnosis Protocol v1 Web
+
+# Logo used for the favicon
+logoPath: './src/assets/img/CowProtocol-icon.svg'
+
+# HTML template
+templatePath: './src/html/index.html'
+
+# Whitelisted tokens from Gnosis Contract retrieved from a smart contract
+tcr:
+  type: 'multi-tcr' # choices: multi-tcr | none
+  config:
+    lists:
+      - networkId: 1
+        listId: 1
+        # contractAddress: '0x1854dae560abb0f399d8badca456663ca5c309d0' # Testing TCR with rare tokens
+        contractAddress: '0x93DB90445B76329e9ed96ECd74e76D8fbf2590d8'
+
+      - networkId: 4
+        contractAddress: '0xBb840456546496E7640DC09ba9fE06E67C157E1b'
+
+# Price estimation/orderbook graph
+dexPriceEstimator:
+  type: 'dex-price-estimator' # choices: dex-price-estimator
+  config:
+    - networkId: 1
+      url_production: https://dex-price-estimator.gnosis.io
+      url_develop: https://price-estimate-mainnet.dev.gnosisdev.com
+
+    - networkId: 4
+      url_production: https://dex-price-estimator.rinkeby.gnosis.io
+      url_develop: https://price-estimate-rinkeby.dev.gnosisdev.com
+
+    - networkId: 100
+      url_production: https://dex-price-estimator.xdai.gnosis.io
+      url_develop: https://price-estimate-xdai.dev.gnosisdev.com
+      
+# Subgraph abstraction, used for getting the last price
+theGraphApi:
+  type: 'the-graph' # choices: the-graph
+  config:
+    - networkId: 1
+      url: https://api.thegraph.com/subgraphs/name/gnosis/protocol
+
+    - networkId: 4
+      url: https://api.thegraph.com/subgraphs/name/gnosis/protocol-rinkeby
+
+    - networkId: 100
+      url: https://api.thegraph.com/subgraphs/name/gnosis/protocol-xdai
+
+# Eth node config
+defaultProviderConfig:
+  type: 'infura' # Choices: infura | url
+  config:
+    # It'll be appended to `infuraEndpoint`
+    infuraId: e941376b017d4dada26dc7891456fa3b
+    infuraEndpoint: wss://mainnet.infura.io/ws/v3/
+  #
+  # Example for type `url`
+  # type: 'url'
+  # config:
+  #   ethNodeUrl: <local eth node>
+
+# Exchange contract config
+exchangeContractConfig:
+  type: 'contractBlock' # choices: contractBlock
+  config:
+    - networkId: 1
+      blockNumber: 9340147
+    - networkId: 4
+      blockNumber: 5844678
+    - networkId: 100
+      blockNumber: 11948310
+
+# Wallet Connect
+walletConnect:
+  bridge: 'wss://safe-walletconnect.gnosis.io/'
```

Comment:
outdated, this is using v1 and a bridge that doesn't exist any more

Reply:
Addressed here: https://github.com/cowprotocol/cowswap/pull/3593